Blueprints/BlueprintDefinitions/vs2022/AspNetCoreWebAPI-FSharp/template/test/BlueprintBaseName.1.Tests/ValuesControllerTests.fs

```diff
@@ -25,6 +25,3 @@ module ValuesControllerTests =
         Assert.True(response.MultiValueHeaders.ContainsKey("Content-Type"))
         Assert.Equal("application/json; charset=utf-8", response.MultiValueHeaders.Item("Content-Type").[0])
     }
-
```

Comment:
why did you remove this bit? (not sure what it does)

Reply:
That is part of that change I mentioned in the description about having FSharp test projects auto generate the entry point.